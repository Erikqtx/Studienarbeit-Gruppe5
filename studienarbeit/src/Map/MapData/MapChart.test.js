import React from 'react';
import ReactDOM from 'react-dom';
import MapChart from "./../MapChart";
import "@testing-library/jest-dom";
import renderer from 'react-test-renderer';

it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<MapChart/>, div)
})

it("matches the snapshot", () => {
    const tree = renderer.create(<MapChart/>).toJSON();
    expect(tree).toMatchSnapshot();
})