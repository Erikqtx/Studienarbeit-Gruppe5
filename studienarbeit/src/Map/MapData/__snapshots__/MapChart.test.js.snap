// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<svg
  className="rsm-svg "
  viewBox="0 0 800 600"
>
  <g
    className="rsm-geographies "
  />
</svg>
`;
